text/069-coq-roadmap.md

```diff
@@ -183,14 +183,54 @@ Would require work on:
 
 #### Promote the Coq Platform
 
-- Théo Zimmermann, ???
+The Coq Platform is already the officially recommended installation method.
+We would like to make it clear to users that they are encouraged to rely on
+the packages that it provides, and that libraries in the Platform should be
+generally considered as a collaborative extended standard library for Coq
+(the historical stdlib being only a part of this).
 
-Editorial work + documentation
+As part of this effort we should do:
+
+- Editorial work for curation of packages included in the Coq Platform
+  (to bring guarantees on their level of quality).
+
+- Consolidation of documentation, so that it is easy for users to find
+  documentation about the included packages, ideally with a consistant
+  presentation.
```

Comment:
Would be great if opam-installing a package could include its documentation and coqdoc (whether the package is in the platform or not) and then have an easy way to access the doc from the GUI.

Also look at ways to enhance coqdoc to make it more useful.